.github/workflows/ci.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         with:
           fail_ci_if_error: true
           verbose: true
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Why we need this?

Reply:
I'm newbie to codecov-action, 
In order to pass the tests,like  https://github.com/honglei/aioftp/actions/runs/5453748546 
I followed : https://github.com/codecov/codecov-action/blob/main/README.md

Reply:
I think `codecov` updates should be excluded in this PR and reimplemented is separate one if someone will need this at all.

Reply:
Coverage working right now as is https://camo.githubusercontent.com/8631edb27e25ba8d58ab1df19fafe2f57991ce73cbe0098f3789f25270feba22/68747470733a2f2f636f6465636f762e696f2f67682f61696f2d6c6962732f61696f6674702f6272616e63682f6d61737465722f67726170682f62616467652e737667

Reply:
https://app.codecov.io/gh/honglei/aioftp/commits

Reply:
Hm, https://app.codecov.io/gh/aio-libs/aioftp

Reply:
I think, there are no pipelines for this PR until I approve it. https://app.codecov.io/gh/aio-libs/aioftp/pull/165